docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,406 @@
+
+Enterprise, dashboards applications, and design systems still continue to be in the top 3, similar to last year. The slight changes though are developers using MUI more to build e-commerce apps and portfolios, which is pretty cool. We'd love to see them, feel free to share in the ["Who's using Material UI?" issue](https://github.com/mui/material-ui/issues/22426).
+
+### What delivery mechanisms are you using?
+
+<img src="/static/blog/2021-survey/19.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Single-page app (Create React App, etc): 74.7%; Server-side rendered website (Next.js, Gastby, etc): 20.8%; Desktop app (Electron, etc): 3.5%; Native mobile app (Cordova, etc): 0.6%; Other: 0.4%" />
+
+<p class="blog-description">1509 out of 1589 answered</p>
+
+### What type system are you using?
+
+<img src="/static/blog/2021-survey/20.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="TypeScript: 63.8%; None: 18%; prop-types: 16.6%; Flow: 1.4%; Other: 0.2%" />
+
+<p class="blog-description">1501 out of 1589 answered</p>
+
+Very different from last year, Typescript just jumped off and it's the primary type system used out there. It seems reasonable to expect more people to use a type system rather than not.
+
+### Which framework are you using, if any?
+
+<img src="/static/blog/2021-survey/21.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Create React App: 62.4%; Next.js: 21.9%; Custom webpack: 10.7%; None: 2.6%; Gatsby: 0.9%; Other: 1.5%" />
+
+<p class="blog-description">1497 out of 1589 answered</p>
+
+A considerable bump in Next.js usage, which is not surprising due to the popularity and advancements it gained in 2021.
+
+### What styling solution are you using?
+
+<img src="/static/blog/2021-survey/22.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="MUI Core v4 (JSS): 45%; Styled components: 37.9%; Emotion: 30.2%; SASS: 20.8%; CSS modules: 18.9%; Vanilla CSS: 17.6%; Tailwind CSS: 9.1%; Stitches: 0.4%; Other: 1.2%" />
+
+<p class="blog-description">1492 out of 1589 answered</p>
+
+As the MUI Core v5 release is still relatively recent, it's still expected to see a great number of developers using JSS, and there will always probably be some of you who use it because of pure preference.
+
+However, it’s notable the styled-components and emotion growth, and that might be because it’s what our new styling solution is based off. We're still seeing a lot of questions and also opportunities to make it better, so this year we'll focus on expanding and refining [MUI System](https://mui.com/system/basics/) even further.
```

Comment:
```suggestion
But the growth of Emotion and styled-components here is noteworthy, because they are what our new styling solution is based on. This year we'll be focusing on expanding and refining [MUI System](https://mui.com/system/basics/) to further improve upon this new styling solution.
```